apps/settings/js/config/require.js

```diff
@@ -380,6 +380,10 @@ require.config({
       exclude: ['main']
     },
     {
+      name: 'modules/jszip',
```

Comment:
Module declaration is used in the optimization stage for r.js to merge all dependent scripts into one starting from the path specified in the name property. Because jszip is self-contained and not an AMD module, we should define a shim for it instead:
```
'modules/jszip': {
  exports: 'JSZip'
}
```